### ISSUE TITLE
AVT 1 Skeleton: Toggle & ToggleSmall DAP violations for Label is missing descriptive text & form control labels

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4180]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
Go to the React skeleton for Toggle & ToggleSmall
Run DAP and the following issue is found on the on each skeleton component (see screenshots):
<img width="1305" alt="toggle" src="https://user-images.githubusercontent.com/21676914/66068118-9bf1d500-e512-11e9-8523-b428d3d249c4.png">
